scripts/claim-netdata.sh

```
@@ -0,0 +1,239 @@
+    echo ""
+    log_info "Claiming $container_name (hostname: $hostname)..."
+    
+    local cmd="docker exec $container_name netdata-claim.sh -token=$CLAIM_TOKEN -url=$CLAIM_URL"
+    if [[ -n "$ROOMS_ARG" ]]; then
+        cmd="$cmd -rooms=$ROOMS_ARG"
+    fi
+    
+    if $DRY_RUN; then
+        log_warn "DRY RUN: $cmd"
+    else
+        if eval $cmd; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)